maintainer/maintainer.go

```diff
@@ -91,6 +91,10 @@ type Maintainer struct {
 	// It is sent to dispatcher managers during bootstrap to initialize their
 	// checkpointTs.
 	startCheckpointTs uint64
+	redoTsMap         map[node.ID]*heartbeatpb.RedoTsMessage
+	// redoTs is global ts to forward
+	redoTs      *heartbeatpb.RedoTsMessage
```

Comment:
use RedoTsMessage both for maintainer to dispatcher, and dispatcher to maintainer may not be a good choice. It will lead some misleading. You can use two different message to make it more clear.(like RedoHeartbeatMessage and RedoTsMessage)

Besides, if checkpointTs is no need in RedoTsMessage, you can directly remove it.